integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,51 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            member: "[.members[].user.email]"
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupProject"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .__team_id + \"/v/b/li/\" + .id "
+            startDate: if .start_date!=null then .start_date | tonumber / 1000 | todate else "" | todate end
+            endDate: if .due_date!=null then .due_date | tonumber / 1000 | todate else "" | todate end
+            totalIssues: .task_count
+          relations:
+            team: .__team_id
```

Comment:
you should return the team full object?

Reply:
This has been done. The full team is now returned